### DISCUSSION TITLE
Dark mode for web ui?

### DISCUSSION THREAD

**Discussion by joheiska** [LizardByte/community#587] [Sunshine]
### Select Topic Area

Feature Request

### Body

I swear I saw a issue that this was being done by someone, but cannot find it anywhere anymore. Is this still under development? If not, I will have to start seriously contemplating on implementing this. My eyes can't take the sunshine of the white background.

edit: Guess the issue was deleted or something, but found that this is already implemented in the pre-realease.

**Comment by ReenigneArcher** [comment]
This is available in beta

**Reply by ReenigneArcher** [reply]
Also, there's darkreader for anything that doesn't have a native dark mode.
